getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -55,8 +64,11 @@ Scroll down to the section where the Android settings are located:
 In that screen, the path to 3 files needs to be set:
 
 -  The *adb* executable (adb.exe on Windows)
+  - It can usually be found at ``%LOCALAPPDATA%\Android\Sdk\platform-tools\adb.exe``.
 -  The *jarsigner* executable (from JDK 6 or 8)
+  - On Windows, OpenJDK installs to ``%PROGRAMFILES%\ojdkbuild\java-1.8.0-openjdk-1.8.0.232-2\bin``. The exact path may vary depending on the OpenJDK update you've installed.
```

Comment:
That's too specific. It depends on what distribution you used for OpenJDK, what version you used, etc.
It should simply mention that it should be in the `bin` directory of the JDK installation.

Reply:
i don't know how to say that tersely yet be obvious to noobs what the folder would look like, so I think this is a good compromise.